api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -56,7 +58,18 @@
    * @since 0.1.0
    */
   public static Tracer getTracer() {
-    return getInstance().tracer;
+    return getTracer("defaultInstrumentationName", "defaultInstrumentationVersion");
+  }
+
+  /**
+   * Returns a named {@link Tracer} with the provided instrumentation name and version.
+   *
+   * @return registered tracer or default via {@link DefaultTracer#getInstance()}.
+   * @throws IllegalStateException if a specified tracer (via system properties) could not be found.
+   * @since 0.1.0
+   */
+  public static Tracer getTracer(String instrumentationName, String instrumentationVersion) {
```

Comment:
I think this class should just return the Factory instance. Maybe in a different PR. The main idea is that this class just manages the global instance of the Factory.

Reply:
In general, I like having the convenience method here, but I do see that with Meters, it would be super unwieldy, super fast. I think it would be fine to expose the factory directly.

Reply:
removed the extra methods.